File: openbis/source/java/ch/systemsx/cisd/openbis/public/resources/api/v3/dto/common/fetchoptions/EmptyFetchOptions.js
define([ "stjs", "dto/common/fetchoptions/FetchOptions" ], function(stjs, FetchOptions) {
	var EmptyFetchOptions = function() {
		FetchOptions.call(this);
	};
	stjs.extend(EmptyFetchOptions, FetchOptions, [ FetchOptions ], function(constructor, prototype) {
		prototype['@type'] = 'dto.common.fetchoptions.EmptyFetchOptions';
		constructor.serialVersionUID = 1;
		prototype.sortBy = function() {
			return null;
		};
		prototype.getSortBy = function() {
			return null;
		};
	}, {});
	return EmptyFetchOptions;
})

File: openbis_api/source/java/ch/ethz/sis/openbis/generic/asapi/v3/dto/common/fetchoptions/EmptyFetchOptions.java
package ch.ethz.sis.openbis.generic.asapi.v3.dto.common.fetchoptions;

import ch.systemsx.cisd.base.annotation.JsonObject;

@JsonObject("as.dto.common.fetchoptions.EmptyFetchOptions")
public class EmptyFetchOptions extends FetchOptions<Void>
{

    private static final long serialVersionUID = 1L;

    @Override
    public SortOptions<Void> sortBy()
    {
        return null;
    }

    @Override
    public SortOptions<Void> getSortBy()
    {
        return null;
    }

}
